release-team/release-team-removal.md

```diff
@@ -0,0 +1,52 @@
+# Release Team Removal Process
+	
+The release team is reassembled with a new team for each cycle. All team members are automatically removed if they do not re-apply and are selected for the next release cycle. While this process generally functions smoothly, there are instances when a team member might leave mid-cycle or the release team leaders may decide to part ways early. 
+
+## Resignation
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emeritus Advisor (EA) to find a replacement if they would like to step down.
+
+
+Remaining on the team without contributing creates an unhealthy environment for the rest of the team and negatively affects the outcome of the release. We strive to be inclusive and empathetic to individuals' circumstances.
```

Comment:
```suggestion
Remaining on the team without contributing creates an unhealthy environment for the rest of the team and negatively affects the outcome of the release. We strive to be inclusive and empathetic to individuals' circumstances, but our first priority is a stable, high-quality release.
```

Adding a bit at the end here just because it feels a touch abrupt without.

Reply:
👍